src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	public static final int INITIAL_RHYMER_INDICATOR = 1;
+	public static final int MAX_SIZE = 12;
+	public static final int IF_EMPTY = 11;
+	private final int[] numbers = new int[MAX_SIZE];
+
+	public int total = -INITIAL_RHYMER_INDICATOR;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?
This constant should be different (separate) than the one in peekaboo and countOut methods.